Documentation/entity-tools.md

```diff
@@ -1,60 +1,68 @@
-# There are few new commands availiable in xash3d fork:
+# Entity tools
+
+For some features described below, you need to enable [console scripting](extensions/console-scripting.md) with command `cmd_scripting 1` in console.
```

Comment:
I wonder if `entity-tools.md` should be in extensions folder too. This feature is kinda unique to us.

Meanwhile, the root `Documentation` folder is getting a bit cluttered. I will work on this later.

Reply:
If you're gonna decide to move it, keep it as a symlink, or just make a new file with a link, in case if anybody using the direct URL somewhere